wordpress-mobile/WordPress-Android
# ExPlat: Remove A/B experiment

This PR removes the A/B experiment added in #14286, as it has already been completed. 

The code changes should be pretty straightforward.

## To test

Make sure all existing unit tests pass. Additionally:

1. Open the app and make sure you've selected a site that supports Stories.
1. On the My Site screen, tap the FAB.
1. Notice that the menu actions have the following order: "Blog post", "Site page" and "Story post".

## Regression Notes
1. Potential unintended areas of impact
Main creation bottom sheet.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
None.

PR submission checklist:
